bottles/frontend/ui/loading.blp

```diff
@@ -15,6 +15,17 @@ template LoadingView : .AdwBin {
         vexpand: true;
       }
 
+      Button btn_go_offline {
+        valign: center;
+        halign: center;
+        label: _("Skip and go offline");
```

Comment:
Minor nit: this should be more in style with the rest of GNOME with regards to capitalization.

I, personally, would prefer "Continue Offline" here.

Reply:
Will "Continue in Offline Mode" a good placement here?

Reply:
I don't think so. "Offline Mode" is unnecessary - just "Offline" works.